plugins/plogs.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: plogs
+spec:
+  version: v0.1.1
+  homepage: https://github.com/kha7iq/plogs
+  shortDescription: Retrieve and stream colorized logs from Kubernetes pods
+  description: |
+    The plogs plugin enables users to retrieve 
+    colorized logs from Kubernetes pods with 
+    various filtering and monitoring options.
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/kha7iq/plogs/releases/download/v0.1.1/plogs_Darwin_x86_64.tar.gz
+    sha256: ff034bee61c66a0bec7dbdf395e7de7f6876dd1e9e775c382e11e460b01bf194
+    bin: plogs
+    files:
+    - from: plogs
+      to: .
+    - from: LICENSE
+      to: .
```

Comment:
You can actually omit the `files` property entirely to extract everything.

Reply:
Done